js/js-function-challenge/js-function-challenge.js

```diff
@@ -0,0 +1,41 @@
+function moonOrbits(days) { 
+    return days / 27;
+}
+console.log(moonOrbits(54));
+
+function circleArea(radius) { 
+    return Math.PI * (radius ** 2);
+}
+console.log(circleArea(10));
+
+function circlePerimeter(radius) { 
+    return 2 * Math.PI * radius;
+}
+console.log(circlePerimeter(10));
+
+function ageInDays(age) { 
+    return age * 365;
+}
+console.log(ageInDays(32));
+
+function ageInSeconds(age) { 
+    return age * 365 * 24 * 60 * 60;
+}
+console.log(ageInSeconds(32));
+
+function remainder(num1, num2) { 
+    return num1 % num2;
+}
+console.log(remainder(1, 3));
+
+function totalPoints(pointer2,pointer3) { 
```

Comment:
Is this supposed to return true or false based on if the total points are less than 100.

Reply:
Sorry I miss read the mvp